airgun/entities/contentviewfilter.py

```diff
@@ -0,0 +1,309 @@
+from airgun.entities.base import BaseEntity
+from airgun.entities.contentview import ContentViewEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.contentviewfilter import (
+    ACTIONS_COLUMN,
+    ContentViewFiltersView,
+    CreateYumFilterView,
+    EditYumFilterView,
+)
+
+
+class ContentViewFilterEntity(BaseEntity):
+
+    def create(self, cv_name, values):
+        """Create a new content view filter"""
+        view = self.navigate_to(self, 'New', cv_name=cv_name)
+        view.fill(values)
+        view.save.click()
```

Comment:
perhaps we are missing
```python
    view.flash.assert_no_error()
    view.flash.dismiss()
```

Reply:
Nope, that's on purpose - no flash notifications on filter creation, user gets redirected to filter page and that's it.

Reply:
and there is success message?